lang/csharp/src/apache/main/Generic/GenericDatumReader.cs

```diff
@@ -159,17 +160,17 @@ public object CreateFixed(object reuse)
                     reuse : new GenericFixed(this.schema);
             }
 
-            public byte[] GetFixedBuffer( object f )
+            public byte[] GetFixedBuffer(object f)
             {
                 return ((GenericFixed)f).Value;
             }
         }
 
-        class GenericArrayAccess : ArrayAccess
+        private class GenericArrayAccess : ArrayAccess
```

Comment:
same

Reply:
By default methods are private (technically internal) unless otherwise set by an access modifier.
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers
"Classes, records, and structs declared directly within a namespace (in other words, that aren't nested within other classes or structs) can be either public or internal. internal is the default if no access modifier is specified."